### PR TITLE
[ETHOSN] Update compilation defaults to Ethos(TM)-N78

### PR DESCRIPTION
**Pull request by Leo-arm**

### CODE REVIEW

**Comment by Leo-arm** [comment]
@leandron if you have time

File: tests/python/contrib/test_ethosn/infrastructure.py [hunk 1]
```diff
@@ -334,4 +334,4 @@ def get_ethosn_api_version():
 
 
 def get_ethosn_variant():
-    return os.getenv("ETHOSN_VARIANT_CONFIG", default="Ethos-N77")
+    return os.getenv("ETHOSN_VARIANT_CONFIG", default="Ethos-N78")
```
Comment (Mousius) [inline tests/python/contrib/test_ethosn/infrastructure.py:337]: Does this need to be the full variant descriptor? And can we remove the NPU specific block from `test_python_integration.sh` if this is the default?

**Comment by Leo-arm** [comment]
@Mousius @leandron if you have a chance.

**Review by manupa-arm — APPROVED** [review]
LGTM

**Comment by manupa-arm** [comment]
Thanks! @Leo-arm @Mousius.

**Pull request merged**